docker/docker_official_images/3.8.0/jvm/Dockerfile

```diff
@@ -18,58 +18,54 @@
 
 FROM eclipse-temurin:21-jre-alpine AS build-jsa
 
-USER root
-
-# Get Kafka from https://archive.apache.org/dist/kafka, url passed as env var, for version 3.7.0
-ENV kafka_url https://archive.apache.org/dist/kafka/3.7.0/kafka_2.13-3.7.0.tgz
+# Get Kafka from https://archive.apache.org/dist/kafka, url passed as env var, for version 3.8.0
+ENV kafka_url https://archive.apache.org/dist/kafka/3.8.0/kafka_2.13-3.8.0.tgz
 
 COPY jsa_launch /etc/kafka/docker/jsa_launch
 
 RUN set -eux ; \
-    apk update ; \
-    apk upgrade ; \
     apk add --no-cache wget gcompat gpg gpg-agent procps bash; \
```

Comment:
`gcompat` is a strange inclusion here -- what in this process needs `glibc` compatibility?